prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -431,4 +450,35 @@ object CustomFHIRFunctions {
             mutableListOf(StringType(result))
         }
     }
+
+    /**
+     * Using the [focus] as the date to be changed and the [parameters] as the timezone to change it to
+     * @return a date in the new timezone
+     */
+    fun changeTimezone(
+        focus: MutableList<Base>,
+        parameters: MutableList<MutableList<Base>>?
+    ): MutableList<Base> {
+        if (focus.size != 1) {
+            throw SchemaException("Must call changeTimezone on a Date")
```

Comment:
Change error message to be more descriptive of the check. You also do not check to see if this is a date time.

Reply:
Adjusted the error message, but still did not check for a date time specifically since the incoming type can be either a string, an instant, a datetime, etc, and the exception checking later on will cover cases where it was not a compatible type.